src/migration/1722942338711-UserEmailIndex.ts

```diff
@@ -0,0 +1,19 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class UserEmailIndex1722942338711 implements MigrationInterface {
+  name = 'UserEmailIndex1722942338711'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE INDEX "IDX_user_loweremail" ON "user" ((lower(email)))`,
+    );
+    await queryRunner.query(`DROP INDEX "public"."IDX_user_email"`);
+    await queryRunner.query(`DROP INDEX "public"."user_email_index"`);
```

Comment:
Should we add `if exists` otherwise it will not work locally?

Reply:
added